src/sentry/seer/similarity/utils.py

```diff
@@ -36,16 +36,6 @@
         "ruby",
     ]
 )
-# Existing projects with these platforms shouldn't be backfilled and new projects with these
-# platforms shouldn't have Seer enabled.
-SEER_INELIGIBLE_PROJECT_PLATFORMS = frozenset(
```

Comment:
All project platforms will be considered, however, the event platform check will stay in place.